app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
```

Comment:
Place all arguments in the `__init__` on a new line to improve the overall readability. There should be no arguments on the same line with the opening and closing parenthesis.